__tests__/api.test.js

```diff
@@ -71,6 +71,91 @@ describe('GET /api/reviews', () => {
                 expect(res.body.msg).toBe('Ooops, nothing to see here!')
             })
     });
+
+    test('can filter by category', () => {
+        return request(app)
+        .get(`/api/reviews/?category=dexterity`)
+        .expect(200)
+        .then((reviews) => {
+            reviews.body.forEach(element => {
```

Comment:
You will need to assert the lengths of the arrays for these responses as the forEach may never run if it is empty